src/include/lean/lean.h

```diff
@@ -451,6 +451,19 @@ static inline void lean_dec(lean_object * o) { if (!lean_is_scalar(o)) lean_dec_
 /* Just free memory */
 LEAN_EXPORT void lean_dealloc(lean_object * o);
 
+/*
+An object `o` satisfies `lean_is_ctor` if it is a member of an inductive type, barring the exceptions
+listed at https://lean-lang.org/lean4/doc/dev/ffi.html?highlight=ffi#translating-types-from-lean-to-c.
```

Comment:
```suggestion
listed at https://lean-lang.org/lean4/doc/dev/ffi.html#translating-types-from-lean-to-c.
```
The highlight parameter was leading to "sufficient" having a highlight, which I don't think we want here.